### ISSUE TITLE
confusing workflow for trading newish coins with already-existing markets

### ISSUE THREAD

**Issue by keone** [project-serum/serum-dex-ui#121]
I'm trying to trade a newish coin, SUNNY (mint=SUNNYWgPQmFxe9wTZzNK7iPnJ3vYDrkgnxJRJm1s3ag).

I can see it's trading quite a bit of volume, e.g. it appears on the top lists of https://coinmarketcap.com/exchanges/serum-dex/ and https://serumpulse.com/main/home

I can even do trades using symmetry.fi's swap interface (which says it just routes to Serum) so I know there's a market for it!

However, it doesn't show up in the market dropdown (the one that defaults to SOL/USDC).  Nor is it searchable (either as SUNNY or as the mint address) in the search to the left of the environment selector.

If I try to "add market" it seems like I can do that, but it will charge me ~2 SOL to create the market (which I don't want to do, the market already exists).

**Comment by dahifi** [comment]
You must be hitting the wrong 'add market' button, you're not creating a market. You don't want the token mint, you want the existing market ID. That doesn't cost anything to add to the plus sign next to the drop down list.

**Comment by keone** [comment]
I agree that if I click the + button to the left of the dropdown it prompts me for a market id... but is there a way to look up the SUNNY/USDC market id?

I mean this is a UX ticket... pretend you are an average user, go to https://dex.projectserum.com/ and try to trade SUNNY/USDC... how do you do it?
